code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -41,6 +41,12 @@
 		reagents.add_reagent(/datum/reagent/toxin/xeno_neurotoxin, reagent_amount)
 		if(prob(10 * S.strength)) //Likely to momentarily freeze up/fall due to arms/hands seizing up
 			to_chat(src, "<span class='danger'>You feel your body going numb and lifeless!</span>")
+	if(CHECK_BITFIELD(S.smoke_traits, SMOKE_XENO_HEMODILE))
+		var/reagent_amount = 5 + S.strength * 2
+		reagents.add_reagent(/datum/reagent/toxin/xeno_hemodile, reagent_amount)
+	if(CHECK_BITFIELD(S.smoke_traits, SMOKE_XENO_TRANSVITOX))
+		var/reagent_amount = 5 + S.strength * 2
+		reagents.add_reagent(/datum/reagent/toxin/xeno_transvitox, reagent_amount)
```

Comment:
No need to declare the variable just put the calculation directly in the proccall

Reply:
changed

Reply:
removed var